README.md

```diff
@@ -34,16 +34,15 @@ make services-invoke-publish
 ```sh
```

Comment:
could we combine `Usage` heading and `Dev setup` in the readme? it's confusing which one to go to